dashboard/test/lib/projects_list_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class ProjectsListTest < ActionController::TestCase
+  CHANNEL_ID = 'STUB_CHANNEL_ID'
+
+  setup do
+    @student = create :student
+
+    student_project_value = '{"name":"Bobs App","level":"/projects/applab","createdAt":"2017-01-24T16:41:08.000-08:00",'\
+      '"updatedAt":"2017-01-25T17:48:12.358-08:00"}'
+    @student_project = {id: 22, value: student_project_value}
+
+    hidden_project_value = '{"name":"Hidden App","level":"/projects/playlab","createdAt":"2017-01-01T00:00:00.000-08:00",'\
```

Comment:
You can construct pretty much anything there, but it's generally more useful for reusable models. These look like arbitrary JSON strings, that have no dependencies and are easy enough to construct directly. Your call, but I'd lean toward leaving them inline in the test.